### PR TITLE
Adds a basic multiZ atmos solution.

### PR DESCRIPTION
**Pull request by Kmc2000**
## About The Pull Request

This PR adds in a multiZ atmos solution. It's really simple and just requires that you line it up. I'm porting this from my downstream because I think it'll encourage people to explore what mul/tg/Z has to offer

It's literally just a pipe, so make sure it has an in bit and an out bit. In my example I hook a connector to the end so atmos techs can siphon off bits of air bound for the next deck up.

This also travels down, and can be used for scrubbers!

Test cases:
Plasmaflood: Succeeded
Drain station of air: Succeeded. Air went into a tank
Flood everything with too much oxygen: Suceeded

This is mapper only until someone can be bothered to make a proper multiZ map.

## Changelog
:cl: Kmc2000
add: Added a multiZ atmos pipe.
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/machinery/pipes/multiz.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	add_overlay(multiz_overlay_node)
+
+/obj/machinery/atmospherics/pipe/simple/multiz/pipeline_expansion() //When new pipes are added to a network, this gets called. Nodes is all the pipes this one's connected to. We bruteforce add the pipes above and below us.
+	var/turf/T = get_turf(src)
+	var/obj/machinery/atmospherics/pipe/simple/multiz/above = locate(/obj/machinery/atmospherics/pipe/simple/multiz) in(SSmapping.get_turf_above(T))
+	if(above)
+		nodes += above
+		above.nodes += src //Two way travel :)
+		return ..()
+	else
```
Comment (ShizCalev) [inline code/modules/atmospherics/machinery/pipes/multiz.dm]: remove the else and just return ..()

**Comment by Razharas** [comment]
Theres entire branch with multiz shit somewhere in the rrs github fork,including my multiz vines and explosions, if you want to revive it find this its like 50% done

**Comment by kevinz000** [comment]
doing god's work

**Comment by AnturK** [comment]
@Kmc2000 Cmon.

**Comment by ExcessiveUseOfCobblestone** [comment]
> losing your pr to else return ..()

yikes

**Pull request closed without merging**